frontend/src/components/Displays/MissionDisplays/MissionAreaDispaly.tsx

```diff
@@ -1,32 +1,23 @@
 import { Typography } from '@equinor/eds-core-react'
-import styled from 'styled-components'
+import { tokens } from '@equinor/eds-tokens'
 import { useLanguageContext } from 'components/Contexts/LanguageContext'
 import { Mission } from 'models/Mission'
-import { tokens } from '@equinor/eds-tokens'
+import styled from 'styled-components'
 
-const StyledRobotDisplay = styled.div`
+const StyledAreaDisplay = styled.div`
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    overflow: hidden;
 `
-const EllipsisTypography = styled(Typography)`
-    overflow: hidden;
-    text-overflow: ellipsis;
-`
-
-interface MissionProps {
-    mission: Mission
-}
 
-export const MissionRobotDisplay = ({ mission }: MissionProps) => {
+export const MissionAreaDisplay = ({ mission }: { mission: Mission }) => {
     const { TranslateText } = useLanguageContext()
     return (
-        <StyledRobotDisplay>
+        <StyledAreaDisplay>
             <Typography variant="meta" color={tokens.colors.text.static_icons__tertiary.hex}>
```

Comment:
Consider using same styling on both robot status and mission status text for consistency and alignment of the items within.
```suggestion
            <Typography variant="meta" style={{ fontSize: 14, color: tokens.colors.text.static_icons__secondary.hex }} >
```